src/components/volume-map-seekbar/volume-map-seekbar.tsx

```diff
@@ -83,21 +83,32 @@ function processVolumeMap(originalMap: VolumeMapEntry[], maxBars: number): Volum
   // Case 3: We need to increase the number of data points through interpolation
   const processedMap: VolumeMapEntry[] = [];
   
-  // Calculate time range of the audio
-  const startPts = originalMap[0].pts;
+  // Calculate time range of the audio, using 0 as the starting point if needed
```

Comment:
see comment in other PR
this is a very long function and i think it should be in a separate file